constraints.txt

```diff
@@ -1,27 +1,35 @@
 ansible==2.2.2.0
+apache-libcloud==1.5.0
 appdirs==1.4.3
 asn1crypto==0.22.0
 Babel==2.4.0
+backports.ssl-match-hostname==3.5.0.1
 cffi==1.10.0
 click==6.7
 configparser==3.5.0
 cryptography==1.8.1
 debtcollector==1.13.0
+defusedxml==0.5.0
 enum34==1.1.6
-flake8==3.3.0
+extras==1.0.0
+fixtures==3.0.0
+flake8==3.4.1
 funcsigs==1.0.2
+gitdb2==2.0.2
 github3.py==0.9.6
+GitPython==2.1.5
 idna==2.5
 influxdb==4.1.1
 ip-associations-python-novaclient-ext==0.2
 ipaddress==1.0.18
 iso8601==0.1.11
-jenkins-job-builder==2.0.0.0b1
+jenkins-job-builder==2.0.0.0b2
```

Comment:
Do we need to do further testing here since this is updating?

Reply:
Good plan, we should build one of the periodic AIOs with b2 to check.